bundles/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/handler/SqueezeBoxServerHandler.java

```diff
@@ -587,6 +597,11 @@ public void updateListener(SqueezeBoxPlayerEventListener listener) {
                     sendCommand(player.getMacAddress() + " status - 1 subscribe:10 tags:yagJlNKjc");
                 }
             }
+            for (final SqueezeBoxPlayer player : players.values()) {
+                final String mac = player.getMacAddress();
+                final PlayerConnectionState state = playerStates.getOrDefault(mac, PlayerConnectionState.OFFLINE);
+                updatePlayer(listener -> listener.connectedStateChangeEvent(mac, state));
+            }
```

Comment:
Not sure whether I should notify of offline status for things that are not in the `players` map here. This isn't needed for the initial case (which works because `connectionState` is initialized to offline in the player handler), but might be needed for cases where we lose connection to the server and a player disappears from the list in the time between our disconnection and reconnection.